lib/descripting/cog.cpp

```diff
@@ -0,0 +1,17 @@
+/* \file
```

Comment:
Doxygen is very picky in this regard (I've tried to change the behavior in the past, never worked), it expects two stars for the first line `/** \file`. Otherwise, it ignores the comment.